mysql-test/t/audit_log_acc.test

```diff
@@ -0,0 +1,71 @@
+# Bug 1641910: Trying to set audit_log_exclude_accounts crashes server.
+
+call mtr.add_suppression("Plugin 'audit_log' init function returned error.");
+call mtr.add_suppression("Plugin 'audit_log' registration as a AUDIT failed.");
+
+--exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
```

Comment:
Here and everywhere please use include/restart_mysqld.inc, it supports restart parameters.